widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -300,7 +300,7 @@ export function Quote(props: QuoteProps) {
     GAS_FEE_MAX_DECIMALS
   );
 
-  return config?.variant === 'full-expanded' && type === 'list-item' ? (
+  return fullExpandMode && type === 'list-item' ? (
```

Comment:
I think we can remove `&& type === 'list-item'`. Am I wrong?

Reply:
Fixed